server/handlers/clients.js

```diff
@@ -159,13 +152,14 @@ async function getConfigHandler(req, res) {
   }
 
   const info = {
-    configs: configurations.data
+    ...getClientInfo(config),
+    configs: config.data
   };
 
   if (req.query.health) {
     try {
       logger.info(`Checking status of client "${req.params.id}"...`);
```

Comment:
lazy render strings in logger `logger.info('foo bar %s', str)`

Reply:
done